protocol/x/clob/keeper/untriggered_conditional_orders.go

```diff
@@ -323,21 +323,19 @@ func (k Keeper) MaybeTriggerConditionalOrders(ctx sdk.Context) (triggeredConditi
 	return triggeredConditionalOrderIds
 }
 
-// CountUntriggeredSubaccountOrders will count all untriggered orders for a given subaccount that match the provided
-// filter.
-func (k Keeper) CountUntriggeredSubaccountOrders(ctx sdk.Context,
+// CountUntriggeredSubaccountStatefulOrders will count all untriggered stateful orders for a given subaccount.
+func (k Keeper) CountUntriggeredSubaccountStatefulOrders(ctx sdk.Context,
 	subaccountId satypes.SubaccountId,
-	filter func(types.OrderId) bool,
 ) uint32 {
 	count := uint32(0)
 	for _, untriggeredConditionalOrders := range k.UntriggeredConditionalOrders {
 		for _, order := range untriggeredConditionalOrders.OrdersToTriggerWhenOraclePriceGTETriggerPrice {
-			if order.OrderId.SubaccountId == subaccountId && filter(order.OrderId) {
+			if order.OrderId.SubaccountId == subaccountId && order.IsStatefulOrder() {
```

Comment:
When will `order.IsStatefulOrder()` ever be false? Short-Term orders can't be conditional

Reply:
I removed it.

It was mainly to prevent a regression in case conditional orders are expanded to include short-term orders for some reason.